lib/3scale/api/client.rb

```diff
@@ -242,6 +242,16 @@ def create_application_plan(service_id, attributes)
         extract(entity: 'application_plan', from: response)
       end
 
+
+      # @api public
+      # @return [Bool]
+      # @param [Fixnum] service_id Service ID
+      # @param [Fixnum] application_plan_id Application Plan ID
+      def delete_application_plan(service_id,application_plan_id)
+        response = http_client.delete("/admin/api/services/#{service_id}/application_plans/#{application_plan_id}")
```

Comment:
`response` is not used

Reply:
Removed the response variable from method.